cmd/cluster/context.go

```
@@ -1003,6 +1110,37 @@ func printDynatraceResources(data *contextData, w io.Writer) {
 	}
 }
 
+func printRhobsResources(data *contextData, w io.Writer) {
+	var name string = "RHOBS Details"
```

golangci-lint (2.12.2)

[error] ST1023: should omit type string from declaration; it will be inferred from the right-hand side

(staticcheck)